Automattic/jetpack
# Woo Analytics: prevent accessing undefined wp.hooks on checkout

## Proposed changes:

* Checks for the presence and type of `wp.hooks.addAction` before accessing it

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Ensure WooCommerce analytics is enabled
* Add a coupon to your site.
* Add an item to your cart and go to the shortcode checkout.
* Ensure no JS errors occur in the console
* Add the coupon and check out.
* Ensure the order is placed successfullly
